src/lib/webbrowser.js

```diff
@@ -12,6 +12,22 @@ var $builtinmodule = function(name){
         window.open(url, "_blank");
 
         return Sk.builtin.bool.true$;
+    }
+
+    mod.open = new Sk.builtin.func(function open(url) {
+        Sk.builtin.pyCheckArgsLen("open", arguments.length + 1, 1, 3);
+        return open_tab(url);
+    });
+
+    mod.open_new = new Sk.builtin.func(function open_new(url) {
+        Sk.builtin.pyCheckArgsLen("open_new", arguments.length + 1, 1, 3);
+        return open_tab(url);
+    });
+
+    mod.open_new_tab = new Sk.builtin.func(function open_new_tab(url) {
+        Sk.builtin.pyCheckArgsLen("open_new_tab", arguments.length + 1, 1, 3);
```

Comment:
for `open` I think it should be 
```javascript
Sk.builtin.pyCheckArgsLen("open", arguments.length, 1, 3); 
// min 1 arg up to 3 args (even though we ignore the other 2)
```
and for the `open_new/open_new_tab` functions - looking at python's call signature - it should be
```javascript
Sk.builtin.pyCheckArgsLen("open_new", arguments.length, 1, 1); 
// exactly 1 arg
```